matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/summary/RoomSummaryConstants.kt

```diff
@@ -32,6 +32,5 @@ object RoomSummaryConstants {
             EventType.CALL_ANSWER,
             EventType.ENCRYPTED,
             EventType.STICKER,
-            EventType.REACTION
```

Comment:
I would merge this PR if this was the only change (with the changelog). Let's do not touch `DisplayableEventFormatter.kt` and `strings.xml`, just in case this is added again (in the project or in a fork).